src/_pytest/logging.py

```diff
@@ -468,6 +468,20 @@ def _setup_cli_logging(self):
             log_cli_handler, formatter=log_cli_formatter, level=log_cli_level
         )
 
+    def set_log_path(self, fname):
```

Comment:
Please write a docstring here given that this is now public. Also mention that this method is still experimental. 👍

Reply:
Thank you, fixed.